README.md

```diff
@@ -62,6 +62,9 @@ $ nextflow run tron-bioinformatics/tronflow-copy-number-calling -r <RELEASE|BRAN
 Option 2: Download the project and run it as follows...
 
 ```bash
+
+git clone --recurse-submodules git@github.com:TRON-Bioinformatics/tronflow-copy-number-calling.git
```

Comment:
Good point, it would be good to have this in every tronflow repo, but this is obviously very difficult to maintain. Maybe we could have generic instructions when possible in the documentation and keep README files to a minimum. Just thinking aloud... maybe we can discuss in the larger group next week.